pandas/tests/groupby/test_transform.py

```diff
@@ -1138,3 +1138,12 @@ def func(grp):
 
     expected = pd.DataFrame([2, -2, 2, 4], columns=["B"])
     tm.assert_frame_equal(result, expected)
+
+
```

Comment:
can you move to pandas/tests/groupby/test_groupby.py, near the other _shift tests